soh/soh/OTRGlobals.cpp

```diff
@@ -1132,7 +1132,7 @@ extern "C" SoundFontSample* ResourceMgr_LoadAudioSample(const char* path)
         sampleC->unk_bit26 = sample->unk_bit26;
         sampleC->unk_bit25 = sample->unk_bit25;
 
-        sampleC->book = new AdpcmBook[sample->book.books.size() * sizeof(int16_t)];
+        sampleC->book = (AdpcmBook*) malloc(sizeof(AdpcmBook) + sample->book.books.size() * sizeof(int16_t));
```

Comment:
Unsure if we want to malloc 1 s16 if books.size() == 0, otherwise this change passes

Reply:
if the size is zero we are allocating too much memory, so maybe we want something like this...
```suggestion
        sampleC->book = (AdpcmBook*) malloc((sample->book.books.size() < 1 ? sizeof(AdpcmBook) : 0) + sample->book.books.size() * sizeof(int16_t));
```

Reply:
We always want to allocate the `AdpcmBook` struct which contains two s32's for order and npredictors, even if the size of the individual book s16's is zero.